infrastructure/base-images/cloudharness-base/Dockerfile

```diff
@@ -12,7 +12,6 @@ RUN apk add gcc libc-dev g++ python3-dev libffi-dev openssl-dev rust musl-dev ca
 COPY libraries/cloudharness-common /libraries/cloudharness-common
 COPY libraries/client/cloudharness_cli /libraries/client/cloudharness_cli
 
-# RUN pip install -r /libraries/cloudharness-common/requirements.txt
```

Comment:
Why this removed?

Reply:
looked to me like commented code that isn't used anymore and replaced by the pip install statement below it, is it still needed?

Reply:
Yes, requirements.txt is different from the dependencies in setup.py because it's meant to keep stricter dependencies and also to track sub-dependencies. So the best practice in production is to install from requirements.txt before installing the package.
It's also useful to first copy/install requirements.txt before copy/install the rest to make a better use of the docker caching